brink.conf

```
@@ -0,0 +1,5 @@
+BASE_REQUIREMENTS='pip==20.2.4 chevah-brink==0.78.1 paver==1.2.4'
+PYTHON_CONFIGURATION='default@2.7.18.bd461724:sol10@2.7.8.bd461724'
+BINARY_DIST_URI='http://binary.chevah.com/production'
+PIP_INDEX='http://pypi.chevah.com/simple'
+CHEVAH_BUILD_DIR='build-server'
```

I think that that just "build" ... or delete this line and leave the default config.

```suggestion
CHEVAH_BUILD_DIR='build'
```

---

Hmm, I think a better name for that directory is perhaps "env"; what do you think?